packages/safe-core-sdk/tests/safeFactory.test.ts

```diff
@@ -95,6 +96,93 @@ describe('Safe Proxy Factory', () => {
     })
   })
 
+  describe('predictSafeAddress', async () => {
+    it('should fail if there are no owners', async () => {
```

Comment:
I think it is possible to deploy a safe with no owners and initialize them later so not sure if we want to fail in this case

Reply:
https://github.com/safe-global/safe-contracts/blob/main/contracts/proxies/GnosisSafeProxyFactory.sol#L67